frontend/packages/integration-tests-cypress/tests/crud/annotations.spec.ts

```diff
@@ -53,22 +53,26 @@ const nameValueEquals = (row: JQuery<HTMLElement>, name: string, value: string)
   });
 };
 
+const numberOfAnnotationsEquals = (count: number) => {
+  // eslint-disable-next-line cypress/no-unnecessary-waiting
+  cy.wait(2000); // wait a couple of seconds for watcher to update page
```

Comment:
This might be necessary in the end, but I feel like we don't really understand the problem. Were you able to reproduce outside of CI by running just annotations spec in a loop until it fails?

My understanding is that the entire `cy.byTestID('edit-annotations').contains(`${count} annotations`)` statement should be retried until it's true. With our long timeouts, I'd expect it to work (particularly when the screenshot shows the value we expect).

Reply:
I agree, I'll keep looking, but I think this is a workaround until a better solution is understood.